flocx_market/tests/unit/models/test_offer_unit.py

```diff
@@ -0,0 +1,25 @@
+from flocx_market.tests.unit.unit_base_test import UnitBaseTest
+from flocx_market.db.sqlalchemy.offer_api import OfferApi
+import datetime
+
+now = datetime.datetime.utcnow()
+
+
+class TestOfferModelUnit(UnitBaseTest):
+    def test_create_offer(self):
+        o = OfferApi('b9752cc0-9bed-4f1c-8917-12ade7a6fdbe',
+                     'Alice1992', now, 'available',
+                     'Dell380', now, now,
+                     {
+                      "new attribute XYZ": "This is jus.",
+                      "cpu_type": "Intel Xeon", "cores": 16,
+                      "ram_gb": 512, "storage_type": "samsung SSD",
+                      "storage_size_gb": 204}, 11.5)
+        self.assertEqual(o.provider_id, "b9752cc0-9bed-4f1c-8917-12ade7a6fdbe")
+        self.assertEqual(o.creator_id, "Alice1992")
+        self.assertEqual(o.marketplace_date_created, now)
+        self.assertEqual(o.status, "available")
+        self.assertEqual(o.server_id, "Dell380")
+        self.assertEqual(o.start_time, now)
+        self.assertEqual(o.end_time, now)
+        self.assertEqual(o.cost, 11.5)
```

Comment:
I think Lars moved this file to tests/unit/db/sqlalchemy/test_offer_api.py, so there's no need to reintroduce this here.

Reply:
That file is different, calling db method. 
This one is only testing the model creation without communicating to the db

Reply:
This is  identical to https://github.com/CCI-MOC/flocx-market/blob/master/flocx_market/tests/unit/db/sqlalchemy/test_offer_api.py#L23

Reply:
sounds good, it was a confusion on my side then on what needs to go into the unit tests @tzumainn @larsks

Reply:
I will add a bunch of more tests that are similar to the test_offer_api for bids and contracts